lib/celluloid/groups/proactor.rb

```diff
@@ -0,0 +1,56 @@
+require 'thread'
+
+module Celluloid
+  class Group
+    class Proactor < Group
```

Comment:
I'm down with `Spawn` or `Spawner` ... and I'll just hard code the finalizer as purely clearing locals and abandon the pattern I was shooting for, for now. I could see a use for different `Group` classes, not only for `ActorSystem`